File: Java/Assign_Cookies.java
void main(){
            int[] g = {1, 2, 3};
        int[] s = {1, 1};

        Arrays.sort(g);
        Arrays.sort(s);

        int i = 0;
        int j = 0;
        int count = 0;

        while (i < g.length && j < s.length) {
            if (s[j] >= g[i]) {
                count++;
                i++;
                j++;
            } else {
                j++;
            }
        }

        System.out.println(count);
}

File: Python/Assign_Cookies.py
def assign_cookies():
    g = [1, 2, 3]
    s = [1, 1]

    g.sort()
    s.sort()

    i = 0
    j = 0
    count = 0

    while i < len(g) and j < len(s):
        if s[j] >= g[i]:
            count += 1
            i += 1
            j += 1
        else:
            j += 1

    print(count)

if __name__ == "__main__":
    assign_cookies()
